Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -297,6 +297,7 @@ WarpXParticleContainer::DepositCurrent(WarpXParIter& pti,
 
     // Lower corner of tile box physical domain
     // Note that this includes guard cells since it is after tilebox.ngrow
+
```

Comment:
Could you revert this change?
```suggestion
```

Reply:
Fixed in commit 887e48d6e8a64ff6f299fb2b513.